docs/install-create-run.qmd

```diff
@@ -149,37 +129,72 @@ You can also modify these settings on a per-file basis with comments at the top
 
 A full list of configuration settings for Pyright/Pylance is available [here](https://github.com/microsoft/pyright/blob/main/docs/configuration.md).
 
-### Using the VS Code Python debugger {#vsc-debug}
+[vscode]: https://code.visualstudio.com/
+[vscode-shiny]: https://marketplace.visualstudio.com/items?itemName=posit.shiny-python
+[vscode-python]: https://marketplace.visualstudio.com/items?itemName=ms-python.python
+:::
+
 
-The Python extension also adds an excellent Python debugger to Visual Studio Code. Each VS Code project/workspace must indicate how it should be launched for debugging purposes, via a `.vscode/launch.json` file. For Shiny apps, that file should look like this:
+## Create
 
-```json
-{
-  // Use IntelliSense to learn about possible attributes.
-  // Hover to view descriptions of existing attributes.
-  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-  "version": "0.2.0",
-  "configurations": [
-    {
-      "name": "Run Shiny app",
-      "type": "python",
-      "request": "launch",
-      "module": "shiny",
-      "args": ["run", "${file}"],
-      "jinja": true,
-      "justMyCode": true
-    },
-  ]
-}
+The best way to create a new Shiny app is with the `shiny create` command line interface (CLI). This command asks you a series of questions about what kind of app you want to create, and then provides all the boilerplate code you need to get started with a working app.
+
+```bash
+shiny create
 ```
 
-With that file is in place, you can start a debugging session: open your `app.py` file and run the Start Debugger command (`F5`). This will attempt to debug `shiny run <path-to-current-file>`.
+![Running the shiny create command from a terminal](assets/shiny-create.mp4){class="img-fluid shadow rounded"}
+
+::: callout-tip
+### Copy/paste examples
 
-If your project contains a single Shiny app, you can replace the `"${file}"` in the above snippet with the relative path to your `app.py` file. With a hardcoded path like this, you can press `F5` to start debugging your app without having to activate the `app.py` editor tab first.
+If you find an example on this site that you want to run/edit locally, you can use `shiny create -t basic-app` to get a basic app template, and then copy/paste the code from the example into the template.
+:::
 
-Note that in debugging mode, edits to your source code will not result in automatic reloading (i.e. the `shiny run` command will not have `--reload`); instead, you will need to restart your debugging session (`Command+Shift+F5` / `Ctrl+Shift+F5`).
+
+## Run
```

Comment:
This is all copied yeah? I believe you can include the contents of one document in the other so that there's only one place to update. Here's how I did that for a presentation: https://github.com/gshotwell/shiny-algorithm/blob/main/index.qmd#L276 but I think there's an easier way if you're not using reveal.js

Reply:
Oh sorry, you had reviewed `docs/create-run.qmd` which was supposed to be deleted. This is the actual file that gets included in the site (and I applied your other feedback to this file).